sql/amo_prod/fenix_addons_by_client_v1/query.sql

```diff
@@ -0,0 +1,82 @@
+CREATE TEMP function get_fields(m ANY TYPE) AS (
+  STRUCT(
+    m.submission_timestamp,
+    m.client_info.client_id,
+    m.sample_id,
+    m.metrics.string_list.addons_enabled_addons,
+    m.client_info.app_build,
+    m.normalized_country_code,
+    m.client_info.locale,
+    m.normalized_os
+  )
+);
+
+CREATE TABLE
+  amo_prod.fenix_addons_by_client_v1
+PARTITION BY
+  submission_date
+CLUSTER BY
+  sample_id
+AS
+WITH unioned AS (
+  SELECT
+    get_fields(m1).*
+  FROM
+    org_mozilla_fenix.metrics AS m1
+  UNION ALL
+  SELECT
+    get_fields(m2).*
+  FROM
+    org_mozilla_fenix_nightly.metrics AS m2
+  UNION ALL
+  SELECT
+    get_fields(m3).*
+  FROM
+    org_mozilla_fennec_aurora.metrics AS m3
+  UNION ALL
+  SELECT
+    get_fields(m4).*
+  FROM
+    org_mozilla_firefox.metrics AS m4
+  UNION ALL
+  SELECT
+    get_fields(m5).*
+  FROM
+    org_mozilla_firefox_beta.metrics AS m5
```

Comment:
> What are Fenix app_ids?

They're roughly equivalent to the concept of "channel" in desktop Firefox. The new Glean telemetry library sends data to different BQ tables based on the application ID in the build, which also determines how it's listed in the Play Store. What we call "Fenix" internally corresponds to 5 distinct apps in the Play store: Firefox Preview, and Firefox Preview Nightly, and then the three channels for users migrated from Fennec (nightly, beta, and release).

The union here is how we pull data from all channels. The data we show for desktop also includes all release channels.